reservations/admin.py

```diff
@@ -165,56 +165,56 @@ def get_berth_switch_reason(self, obj):
 
     get_berth_switch_reason.short_description = _("Reason")
 
-    def export_reservations(self, request, queryset):
+    def export_applications(self, request, queryset):
         response = HttpResponse(
             content_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
         )
 
         local_datetime_now_as_str = datetime.datetime.now(
             timezone(settings.TIME_ZONE)
         ).strftime("%Y-%m-%d_%H-%M-%S")
-        filename = "berth_reservations_" + local_datetime_now_as_str
+        filename = "berth_applications_" + local_datetime_now_as_str
         response["Content-Disposition"] = "attachment; filename=%s.xlsx" % filename
 
-        response.content = export_berth_reservations_as_xlsx(queryset)
+        response.content = export_berth_applications_as_xlsx(queryset)
         return response
 
-    export_reservations.short_description = _(
-        "Download list of chosen reservations in Excel file"
+    export_applications.short_description = _(
+        "Download list of chosen applications in Excel file"
     )
 
-    def resend_reservation_confirmation(self, request, queryset):
+    def resend_application_confirmation(self, request, queryset):
         resent_count = 0
-        for reservation in queryset:
+        for application in queryset:
             try:
                 send_notification(
-                    reservation.email,
-                    NotificationType.BERTH_RESERVATION_CREATED,
-                    reservation.get_notification_context(),
-                    reservation.language,
+                    application.email,
+                    NotificationType.BERTH_application_CREATED,
```

Comment:
enum members should be fully capitalized per convention

Reply:
Forgot to add `Preserve case` on Pycharm :derp: